src/vivarium/framework/results/manager.py

```diff
@@ -89,12 +89,11 @@ def on_post_setup(self, _: Event):
                     }
                     # Get the complete cartesian product of stratifications
                     index_values = list(itertools.product(*stratification_values.values()))
-                    self._metrics[measure] = pd.DataFrame(
-                        data=0.0,  # Initialize to 0
-                        columns=["value"],
+                    self._metrics[measure] = pd.Series(
+                        data=0.0,
+                        name=measure,
                         index=pd.MultiIndex.from_tuples(
-                            index_values,
-                            names=stratification_values.keys(),
+                            index_values, names=stratification_values.keys()
```

Comment:
Ok higher level question here to make sure I am understanding this correctly - we changed from having a 1 column dataframe for each measure to a series and we store a dict of these series in self._metrics. I assume the reason self._metrics is not a dataframe (wide?) is because me would have different indices for different measures?

Reply:
It really could be either a df or a series. It just feels more natural to be a series where the multi-index are the groups/stratifications and the values are the, well, values. I initially was intending on having a single-col dataframe with that column named "value" but that was just begging to be converted to a series. But it really is the same.

Reply:
After all that, I'm actually switching back to DataFrames